Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -221,6 +227,13 @@ public class LocalStorage {
         }
     }
 
+    static func moveToSubmittedJobs(jobId: String) throws {
+        try createDirectory(at: submittedDirectory, overwrite: false)
+        let unsubmittedFileDirectory = try unsubmittedDirectory.appendingPathComponent(jobId)
+        let submittedFileDirectory = try submittedDirectory.appendingPathComponent(jobId)
```

Comment:
a little pedantic, but for slightly more accurate naming
```suggestion
        let unsubmittedJobDirectory = try unsubmittedDirectory.appendingPathComponent(jobId)
        let submittedJobDirectory = try submittedDirectory.appendingPathComponent(jobId)
```